packages/bot-web-ui/src/assets/sass/notify-item.scss

```diff
@@ -9,6 +9,25 @@
             .btn__text {
                 font-size: var(--text-size-xxs);
             }
+            &-journal {
+                margin-left: 0 !important;
+                transform: none !important;
+                height: 2.4rem !important;
+                margin-top: 0.8rem !important;
+
+                .dc-btn__text {
+                    font-size: 1.2rem !important;
+                }
+                &--mobile {
+                    height: 2.4rem !important;
+                    margin-left: 100% !important;
+                    transform: translateX(-95%) !important;
+
+                    .dc-btn__text {
+                        font-size: 1.2rem !important;
+                    }
+                }
```

Comment:
- Do we have to use `!important` everywhere? If we make the rules more specific these should take precedence over the existing ones.
- Could we use CSS variables for text-sizes?